Dockerfile

```diff
@@ -5,20 +5,28 @@ FROM python:3.8.2
 # set MkDocs version
 ENV MKDOCS_VERSION="1.5.3"
 
-RUN apt-get install -y git && \
-    pip install git+https://github.com/hfagerlund/mkdocs-docskimmer.git@issue-17_fix-config-value
-
-WORKDIR /mkdocs-docskimmer
+WORKDIR /build
 
 RUN pip install mkdocs==${MKDOCS_VERSION} && \
-    mkdocs new my-project && \
-    cd my-project
-
-COPY docs_local-updates docs
+    mkdocs new my-project
 
-RUN mkdocs --version
-
-EXPOSE 80
+RUN apt-get install -y git && \
+    # install latest version (modify to install another tag or branch - eg. ...@v0.3.1)
+    pip install git+https://github.com/hfagerlund/mkdocs-docskimmer.git@master
+
+RUN cd my-project
+COPY docs_local-updates ./my-project/docs
+
+RUN echo 'site_name: docker-for-docskimmer\n\
+theme:\n\
+  name: docskimmer\n\
+  include_search_page: true\n\
+  search_index_only: false\n\
+  static_templates:\n\
+  - 404.html\n\
+plugins: ['search']\n\
+extra:\n\
+    version: 0.4.0'\ >> ./my-project/mkdocs.yml
```

Comment:
* Points pip install to master branch (was previously experimental branch, now deleted since PR was merged/closed);
* Writes mkdocs.yml within the Dockerfile